tests/unit/amo/sagas/test_reviews.js

```diff
@@ -674,6 +676,26 @@ describe(__filename, () => {
       expect(action).toEqual(expectedAction);
     });
 
+    it('does not re-fetch the add-on after updating a reply', async () => {
+      const slug = 'the-addon-slug';
+      mockApi
+        .expects('submitReview')
+        .resolves(
+          createExternalReview({ addonSlug: slug, isDeveloperReply: true }),
+        );
+
+      _updateAddonReview();
+
+      const expectedAction = hideFlashedReviewMessage();
+      await sagaTester.waitFor(expectedAction.type);
+
+      const unexpectedAction = fetchAddon({ errorHandler, slug });
+
+      expect(sagaTester.getCalledActions().map((a) => a.type)).not.toContain(
```

Comment:
can you just do this here:
` expect(sagaTester.numCalled(unexpectedAction.type)).toEqual(0);` ?

Reply:
Agreed, this seems simpler. I was just following the model we use everywhere else. Thanks @rebmullin!